.github/workflows/ci.yaml

```diff
@@ -111,10 +111,10 @@ jobs:
           sudo snap start opensearch.daemon
 
           # wait a bit for it to fully initialize
-          sleep 15s
-
+          sleep 40s
```

Comment:
Interesting that it needs more time to start, now that we have bigger GH runner machines

Reply:
Yes, @Mehdi-Bendriss I am unsure why, but I had to extend it. Btw, I retested the CI 3x times, all 3 came green afterwards.